src/init.c

```diff
@@ -72,6 +73,26 @@ static void reset_callback_register(void)
   );
 }
 
+/** Callback to read STM32F4's hardcoded unique ID.
+ * Sends STM32F4 unique ID (12 bytes) back to host.
+ */
+void stm_unique_id_callback(u16 sender_id, u8 len, u8 msg[], void* context)
+{
+  (void)sender_id; (void)len; (void)msg; (void) context;
+
+  sbp_send_msg(SBP_MSG_STM_UNIQUE_ID, 12, (u8*)STM_UNIQUE_ID_ADDR);
+}
+
+/** Register callback to read Device's Unique ID. */
+void stm_unique_id_callback_register(void)
```

Comment:
If they are just used in that file you can indicate it using the `static` keyword.